css/components/_figure.scss

```diff
@@ -13,5 +13,10 @@ figure {
     border-top: 1px solid $gray--mid;
     padding: 2px 0;
     font-size: 0.8rem;
+
+    &.thick {
```

Comment:
If you feel like you absolutely want to keep the bold font, please use font-weight: 500 instead of font-weight: bold